advanced-usage/concrete-mocking.md

```diff
@@ -1,242 +1,318 @@
 ---
-title: Concrete Mocking
-page_title: Concrete Mocking | JustMock Documentation
-description: Concrete Mocking
+title: Non-Abstract and Non-Virtual
+page_title: Mock Non-Abstract and Non-Virtual Classes or Members | JustMock Documentation
+description: Mock Non-Abstract and Non-Virtual Classes or Members
 previous_url: /advanced-usage-concrete-mocking.html
 slug: justmock/advanced-usage/concrete-mocking
-tags: concrete,mocking
+tags: non-virtual,non-abstract, concrete, mocking
 published: True
 position: 7
 ---
 
-# Concrete Mocking
+# Mock Non-Abstract and Non-Virtual Classes or Members
 
-Concrete mocking is one of the advanced features supported in __Telerik® JustMock__. Up to this point we have been talking mostly about mocking interfaces. This feature allows you to mock the creation of an object. To some extent this is available in the free edition and there are more things you can do in the commercial edition of the product. In this topic we will go through some examples demonstrating these differences.
+The advanced features supported in __Telerik® JustMock__ enables you to mock any class or member, including non-virtual and non-abstract implementations. The mocking of **non-abstract classes** is also available in the [free edition]({%slug justmock/basic-usage/non-abstract%}) but the commercial version also includes **mocking of non-abstract and non-virtual members** of classes. 
```

Comment:
Mocking of non-abstract classes in the free version is very limited and only supports mocking a concrete class with virtual methods. Can we remove this information from this article or update it to reflect what is actually supported? If you decide to update the article you could think about whether to include the example for that scenario from the concrete mocking article.